synapse/static/client/login/js/login.js

```diff
@@ -33,7 +33,7 @@ var submitToken = function(loginToken) {
 };
 
 var errorFunc = function(err) {
-    show_login();
+    show_login(true);
```

Comment:
comment please: why do we inhibit redirect? (so that the user can see the error, presumably)